lib/promoted/ruby/client/options.rb

```diff
@@ -177,14 +177,16 @@ def compact_insertions
           if limit
             insertions_to_compact = insertions_to_compact[0..limit-1]
           end
+          offset = request[:paging][:offset]
+          offset ||= 0
```

Comment:
I'm not sure I see the purpose of this line of code (?).

Reply:
I don't know Ruby.  Deferring to a Ruby expert.

Reply:
I believe the only thing it could do is coalesce nil into 0, in which case I guess I'd write it more clearly with a ternary or something.